gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.daemon=false
 org.gradle.parallel=false
 org.gradle.jvmargs=-Xms512m -Xmx512m
-kafkaVersion=2.4.1
-zookeeperVersion=3.6.0
\ No newline at end of file
+kafkaVersion=2.5.0
+zookeeperVersion=3.6.1
```

Comment:
Can we avoid version bump to `3.6.1`? -- please see https://github.com/linkedin/cruise-control/pull/1236#discussion_r438252027

Reply:
Thanks @efeg - good catch! I missed your previous comment